integration/test/ParseObjectTest.js

```diff
@@ -381,7 +378,7 @@ describe('Parse Object', () => {
     } catch (error) {
       assert.equal(
         error.message,
-        'schema mismatch for TestObject.hello; expected String but got Object'
+        `Cannot create property 'dot' on string 'world'`
```

Comment:
Can you add a separate test to ensure that you can increment on nested array field

Reply:
Done :+1:

Reply:
I mean create a separate integration test. The unit tests looks fine but we would want to see real objects being saved and queried.